test/integration/controller/job/job_controller_test.go

```diff
@@ -38,6 +39,7 @@ import (
 	"sigs.k8s.io/kueue/pkg/util/pointer"
 	"sigs.k8s.io/kueue/pkg/util/testing"
 	testingjob "sigs.k8s.io/kueue/pkg/util/testingjobs/job"
+	testingmpijob "sigs.k8s.io/kueue/pkg/util/testingjobs/mpijob"
```

Comment:
We should have these tests in the mpijob package instead

Reply:
I put these tests on the job package since we haven't yet ways to start both mpijob-controller and job-controller in the integration test.

But, we can put these tests on the mpijob package by expanding `managerSetup`.

Reply:
Anyway, I will do that in this PR.

Reply:
done.